go.mod

```diff
@@ -85,6 +85,7 @@ require (
 	github.com/breml/bidichk v0.2.4 // indirect
 	github.com/breml/errchkjson v0.3.1 // indirect
 	github.com/bufbuild/connect-go v1.8.0 // indirect
+	github.com/bufbuild/connect-opentelemetry-go v0.3.0 // indirect
```

Comment:
From what I can tell, the only place we use buf as a dependency in our code is here: https://github.com/cometbft/cometbft/blob/f23414f59ace69608f2b93c22700959f04beacaa/tools/tools.go#L10

See the `tools/tools.go` file for a description of why that's needed.

As such it doesn't seem like an issue for me, because we only use buf as a tool to generate our protos.

See also the dependency in question for more details on what its purpose is: https://github.com/bufbuild/connect-opentelemetry-go